src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Blogs from "./index";
+
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () =>
+      Promise.resolve({
+        title: {
+          rendered: "Women standing together",
+        },
+      }),
+  })
+);
+
+describe("Blogs", () => {
+  it("renders correctly and matches the snapshot", () => {
+    const tree = renderer.create(<Blogs />).toJSON();
```

Comment:
```
   const fetchedBlogPosts = await data.json();
    setBlogPosts(fetchedBlogPosts);
```

see this line in your code

you want what you return on line 9-11 to be valid data so that it makes your snapshot test show a blog

please ask for another review when the snapshot test shows an example of a blog post (i can tell that you're not even checking your snapshot test because of what it says right now, otherwise you'd know that the test you wrote doesn't work)

the idea here is that you are mocking fetch so that it gives you fake wordpress data which then creates an example of a blog post in the snapshot test

work with your teammates or ammar to get this finished today please. @jennethydyrova @Jamil-Bailony @hiba-machfej @Ammar-64